docker/hephaestus-debug/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Build Hephaestus image
+# Get base image
+FROM alexanderianblair/hephaestus-deps:master
```

Comment:
I think this should be `FROM alexanderianblair/hephaestus-debug-deps:master` here, to make sure the dependencies are built with debug symbols also